pages/Cloth/[...ClothNumber].tsx

```diff
@@ -54,7 +54,7 @@ const Cloth = () => {
     router.push(`/EditCloth/${router.query.ClothNumber![0]}`);
   };
 
-  const [URLState, setURLState] = useState<any>(true);
+  const [URLState, setURLState] = useState<any>(false);
```

Comment:
`true or false` 라면
any보다는 타입을 명시해주는게 좋아보입니다!
```suggestion
  const [URLState, setURLState] = useState<Boolean>(false);
```

Reply:
오...왜 그랬을까요 ㅋㅋㅋ 수정해놓겠습니다

Reply:
수정하였습니다